motion/ruby_motion_query/data.rb

```diff
@@ -64,6 +64,12 @@ def style_name=(value)
       styles[0] = value
     end
 
+    # view.rmq_data.styles = :something
```

Comment:
we need this to be an array right?  Might be worth updating the comment to be `styles = [:something]` to help show that.  We could probably put in a guard too `styles = Array(s)` if we were worried that it would be used in another way.